src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -622,7 +622,7 @@ object LoadVCF {
       case (VCFHeaderLineType.String, false) => TString()
       case (VCFHeaderLineType.Character, false) => TString()
       case (VCFHeaderLineType.Flag, false) => TBoolean()
-      case (_, true) => fatal(s"Can only convert a header line with type `String' to a Call Type. Found `${ line.getType }'.")
+      case (_, true) => fatal(s"Can only convert a header line with type `String' to a call type. Found `${ line.getType }'.")
```

Comment:
Type shouldn't be capitalized. How about:
Header line must have type `String' in order to cast to type Call. Found ...

Reply:
the String refers to the VCF type, which is capitalized

Reply:
I meant the word "Type"

Reply:
I lower-cased Call and Type here. Not seeing what you're seeing, sorry!

Reply:
Oops, I was looking at the old version. Nm!